lxd/api_1.0.go

```diff
@@ -304,6 +306,37 @@ func doApi10Update(d *Daemon, req api.ServerPut, patch bool) Response {
 		if err != nil {
 			return errors.Wrap(err, "Failed to load node config")
 		}
+
+		// We currently don't allow changing the cluster.https_address
+		// once it's set.
+		curClusterAddress := newNodeConfig.ClusterAddress()
+		newClusterAddress, ok := nodeValues["cluster.https_address"]
+
+		clusterAddress := curClusterAddress // Effective cluster address after the change
+
+		if ok {
+			if curClusterAddress != "" {
+				if newClusterAddress != curClusterAddress {
+					return fmt.Errorf("Changing cluster.https_address is currently not supported")
+				}
+			} else {
+				clusterAddress = newClusterAddress.(string)
+			}
+		}
+
+		// Setting a wildcard address for core.https_address is only
+		// allowed if the port differs from cluster.https_address.
```

Comment:
Hmm, no, there's nothing wrong with doing:
 - core.https_address :8443
 - cluster.https_address 192.168.1.10:8443

We just need to be smart and detect that core.https_address covers cluster.https_address and turn off the cluster http handler in that case, then if core.https_address is changed in a way where it no longer covers cluster.https_address we should bring that handler back up.

Reply:
Ok, I've changed code to handle that situation.